src/plugins/inline-popup/inline-popup.ts

```diff
@@ -21,9 +21,16 @@ import {
 } from '../../types';
 import { makeCollection } from '../../modules/toolbar/factory';
 import { Popup } from '../../core/ui/popup';
-import { splitArray, isString, position, isFunction } from '../../core/helpers';
+import {
+	splitArray,
+	isString,
+	position,
+	isFunction,
+	camelCase
+} from '../../core/helpers';
 import { Dom, ToolbarCollection } from '../../modules';
 import { debounce, wait } from '../../core/decorators';
+import { isArray } from 'util';
```

Comment:
../../core/helpers has isArray

Reply:
oh oops, thank you! i will fix that & rebase the PR